blueocean-core-js/src/js/utils/UrlUtils.js

```diff
@@ -274,7 +283,7 @@ export function toClassicJobPage(pageUrl, isMultibranch = false) {
         // The next token is the "full" job name, URL encoded.
         const fullJobName = decodeURIComponent(pageUrlTokens.shift());
         const fullJobNameTokens = fullJobName.split('/');
-        const classicJobFullName = '/job/' + fullJobNameTokens.join('/job/');
+        const classicJobFullName = jobPrefixPath(AppConfig.getOrganizationName()) + fullJobNameTokens.join('/job/');
```

Comment:
This looks wrong to me, it should be that should be the path of the organization group.
https://github.com/jenkinsci/blueocean-plugin/blob/8d8d5234579167ebbc072afda09d554d002ec8b0/blueocean-rest/src/main/java/io/jenkins/blueocean/rest/factory/organization/AbstractOrganization.java#L21

Prepending the path of `organization.getGroup()` should be enough